platform-coop-toolkit/coop-library-framework
# Add publication fields (resolve #114)

## Description

Adds fields for publication name and link (for resources which appear within a publication such as a journal or magazine). Distinct from publisher.

## Steps to test

1. Add and save publication and publication links.

**Expected behavior:** Links are validated, and both fields are saved as expected.

## Additional information

Not applicable.

## Related issues

- Resolves #114
